pcsm/catalog.go

```diff
@@ -795,6 +795,8 @@ func (c *Catalog) Finalize(ctx context.Context) error {
 
 	for db, colls := range c.Databases {
 		for coll, collEntry := range colls.Collections {
+			lg := lg.With(log.NS(db, coll))
```

Comment:
We need a new var here, like `l` or something, because like this we will shadow `lg` from line 790 and then if need to log a error on line 881, than it will have NS from the last loop, but we log errors for all namespaces.

Reply:
Ah yes, that's correct.